newsletter-may-2023.rst

```diff
@@ -0,0 +1,71 @@
+.. post:: May 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+   :category: Newsletter
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - May 2023
+=====================================
+
+News and updates
+----------------
+
+- 🚁️ The proxy application El Proxito that serves all of our documentation websites has a new version that's being slowly rolled out.
+  Several projects are now served using its new URL resolver,
+  which is faster and is receiving new features.
+  We expect to roll it out on all projects soon.
```

Comment:
@stsewd can you help with suggestions or inputs for this item? Is there a specific new feature or optimization that should be mentioned?

Reply:
Custom URLs will be exciting to announce, but I think maybe this announcement is becoming too unspecific then. I'll move it into upcoming work :+1:

Reply:
I noticed afterwards that I had coupled this announcement with the announcement of improved 404s :see_no_evil: 

So it stays, but I made an extra effort to make the whole update easier to understand.